src/app/components/header/header.component.html

```diff
@@ -1,5 +1,5 @@
 
-<mat-toolbar class="mat-elevation-z2 edcs-header" color="primary">
+<mat-toolbar mat-elevation-z2 edcs-header color="primary">
```

Comment:
What is this change for?

Reply:
lol, nothing, it was a mistake I didn't catch